distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -804,62 +810,12 @@ State         Version          Name                          Repository  Descrip
 ----
 
 ==== Configuring {branding} using Configuration (.cfg) files
-The {branding} can also be configured with configuration (.cfg) files.
 
 [IMPORTANT]
 ====
 Since certain bundles can only be configured using the new `.config` file format, this new file format is now recommended. Please refer to the Configuring {branding} using Configuration (.config) Files section for more details.
```

Comment:
Instead of "is now recommended", maybe we should say "should be used". Thoughts?